### DISCUSSION TITLE
Renovate "Migrate Renovate Config" PR is attempting to delete package.json contents

### DISCUSSION THREAD

**Discussion by wopian** [renovatebot/renovate#33078] [Request Help] [answered]
### How are you running Renovate?

A Mend.io-hosted app

### Please tell us more about your question or problem

I clicked the checkbox on my dependencies dashboard issue and a PR was created to migrate the config to non-deprecated versions.

The PR created is replacing the entire package.json (metadata, scripts, dependencies) with just the Renovate config hoisted to the top level. Effectively deleting all the package information.

This appears to be malformed behaviour and should be creating a new file (renovate.json?)

Example PR: https://github.com/zeepkist/zeepkist-legacy/pull/1402

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
That's not intended. I thought we'd skip package.json altogether in fact, but renaming it to renovate.json is a good stretch goal too.

**Comment by rarkins** [comment, marked as answer]
Converted to bug issue: https://github.com/renovatebot/renovate/issues/33116
